### PR TITLE
Descriptive metadata folder on representation level

### PR DESCRIPTION
**Pull request by shsdev**
There should be no descriptive metadata folder on the representation level, see https://github.com/DILCISBoard/E-ARK-AIP/issues/77

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -255,6 +255,10 @@ are in the Open Document Format (ODT) and in the second one - as a derivative
 of the first representation - all files are in the Portable Document Format
 (PDF).
 
+Note that in Figure [4](#fig4) and Figure [5](#fig5) there is no folder for descriptive metadata on the representation level.
+The reason for this is that new representations are added as a new form for persisting and visualising the content. Metadata
+relate to the intellectual entity and should be maintained on the root level.
```
Comment (kieranjol) [inline index.md:260]: Should this be “descriptive metadata”?

**Review by kieranjol — COMMENTED** [review]
Looks good to me aside from one potential typo. Thanks for adding this!

File: index.md (hunk 1 above)
Comment (shsdev) [reply index.md]: You're right, I corrected it.

File: index.md [hunk 2]
```diff
@@ -97,11 +97,19 @@ A new version of an AIP can contain one or more new representations which can be
...
+either metadata of the logical AIP or representation information was changed, 
+i.e. one or more representations have been modified or removed or were added.
 
 If the logical AIP is changed, the physical representation of the information
-in a container may change as well. The result is a new version of the physical
-container files.
+in a container may change as well. 
+
+*Definition:* A *generation* is a manifestation of a logical AIP in form of
+one ore several physical container files.
```
Comment (kieranjol) [inline index.md:112]: Typo, should be “or”

**Pull request merged**